LoopKit/GlucoseKit/GlucoseStore.swift

```diff
@@ -286,6 +286,7 @@ extension GlucoseStore {
         dataAccessQueue.async {
             self.purgeCachedGlucoseObjects(matching: cachePredicate)
             self.healthStore.deleteObjects(of: self.glucoseType, predicate: healthKitPredicate, withCompletion: completion)
+            self.updateLatestGlucose()
```

Comment:
Hm...how is this related to the valid sensor session?

Reply:
This is fixing a separate bug, that I noticed when working on this, where latestGlucose is not cleared out after testing sensor data is purged.

Reply:
I can put this into a separate PR if you feel that's warranted.

Reply:
Ah, nah, not necessary to split out into another PR. :)

FWIW, I'd be fine, next time, if you just added a PR comment before reviewing starts that says, "Yeah, this is a separate [unfiled] bug fix, I just rolled it in here for convenience."